notification_center.py

```diff
@@ -17,22 +22,35 @@
 	'show_highlights': 'on',
 	'show_private_message': 'on',
 	'show_message_text': 'on',
-	'sound': 'off',
+	'show_other_message': 'on',
+	'sound': 'on',
+	'sound_on_other_message': 'off'
 }
 
 for key, val in DEFAULT_OPTIONS.items():
 	if not weechat.config_is_set_plugin(key):
 		weechat.config_set_plugin(key, val)
 
-weechat.hook_print('', 'irc_privmsg', '', 1, 'notify', '')
-
 def notify(data, buffer, date, tags, displayed, highlight, prefix, message):
-	# passing `None` or `''` still plays the default sound so we pass a lambda instead
+
+	#If own nickname is sender, ignore
+	mynick = weechat.buffer_get_string(buffer,'localvar_nick')
+	if prefix == mynick or prefix == ('@%s' % mynick):
+		return weechat.WEECHAT_RC_OK
+
+	#Format message
+	message = repr(message.encode('string-escape'))
```

Comment:
Because if you want a notification for "other messages" it also shows the messages you send.

The format was because the messages I received contained code which messed up pync. You can ignore it.